.gitignore

```diff
@@ -1,4 +1,5 @@
 /venv/
+/.venv/
```

Comment:
I think the `.gitignore` could be autogenerated with `gitignoreio` like most of our other repos. I'll do a PR for it.